{
	"identifier": "CVE-2020-25032",
	"description": "An issue was discovered in Flask-CORS (aka CORS Middleware for Flask) before 3.0.9. It allows ../ directory traversal to access private resources because resource matching does not ensure that pathnames are in a canonical format.",
	"type": "network",
	"severity": "medium",
	"state": "published",
	"datetime": "0000-00-00T00:00:00Z",
	"is_edited": false,
	"hardware": [],
	"software": [
		{
			"name": "python-flask-cors",
			"version": "\u003c 0:3.0.9-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 0:3.0.7-1+deb10u1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 3.0.9-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bookworm"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 3.0.9-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 3.0.7-1+deb10u1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 3.0.9-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-sid"
		},
		{
			"name": "python-flask-cors",
			"version": "\u003c 3.0.9-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-trixie"
		},
		{
			"name": "python-flask-cors",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-focal"
		},
		{
			"name": "python-flask-cors",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-groovy"
		},
		{
			"name": "python3-flask-cors",
			"version": "\u003c 0:3.0.8-2ubuntu0.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-focal"
		}
	],
	"products": [
		{
			"name": "flask-cors",
			"version": "\u003c 3.0.9",
			"architecture": "any",
			"vendor": "flask-cors-project",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "debian-linux",
			"version": "10.0",
			"architecture": "any",
			"vendor": "debian",
			"type": "system",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "backports-sle",
			"version": "15.0-sp1",
			"architecture": "any",
			"vendor": "opensuse",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "backports-sle",
			"version": "15.0-sp2",
			"architecture": "any",
			"vendor": "opensuse",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "leap",
			"version": "15.1",
			"architecture": "any",
			"vendor": "opensuse",
			"type": "system",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "leap",
			"version": "15.2",
			"architecture": "any",
			"vendor": "opensuse",
			"type": "system",
			"is_edited": false,
			"aliases": []
		}
	],
	"references": [
		"http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00028.html",
		"http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00032.html",
		"http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00039.html",
		"http://lists.opensuse.org/opensuse-security-announce/2020-09/msg00048.html",
		"https://github.com/corydolphin/flask-cors/releases/tag/3.0.9",
		"https://www.debian.org/security/2020/dsa-4775",
		"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25032",
		"https://github.com/corydolphin/flask-cors/commit/67c4b2cc98ae87cf1fa7df4f97fd81b40c79b895",
		"https://ubuntu.com/security/notices/USN-6019-1"
	],
	"weaknesses": [
		"CWE-22"
	]
}